surveys/micro/embedded-wg-2024-08.md

```diff
@@ -0,0 +1,139 @@
+# Survey questions
+
+This is the first Rust Embedded Working Group community survey. The goal is to gather feedback from users of Rust on bare-metal or RTOS based embedded devices including microcontrollers.
+
+## Your use of embedded Rust
+
+### Where do you plan to use embedded Rust in the next 12 months?
+
+Type: select all that apply (optional)
+
+- Hobby or personal projects
+- School or university projects
+- Work or professional projects
+- Not currently using embedded Rust, but have in the past for projects
+- I have not used embedded Rust for any projects
+
+### How long have you been using embedded Rust?
+
+Type: select one (optional)
+
+- Zero to one years
+- One to two years
+- Two to three years
+- Three to four years
+- Four to five years
+- Five to six years
+- More than six years
```

Comment:
Do you need all these options or can they be halved (f.e. 0-2, 2-5, 6+)?

(possibly yes considering the ecosystem's young age but asking either way :slightly_smiling_face: )